src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,7 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+
+        return ((a) ? 1 : 0) + ((b) ? 1 : 0) + ((c) ? 1 : 0) + ((d) ? 1 : 0) == 2;
```

Comment:
Main goal while completing this task is to learn how to work with logical operators, your solution is nice, but let's try solve it with &&, || and ^ operators, you will use them a lot in future 🙂

Reply:
i have fixed and created new pull request.  now everything is good?
return ((a ^ b) & (c ^ d)) | ((a ^ c) & (b ^ d)) | ((a ^ d) & (b ^ c));